packages/login/src/vaadin-login-form.js

```diff
@@ -71,11 +71,6 @@ class LoginForm extends LoginFormMixin(ElementMixin(ThemableMixin(PolylitMixin(L
   /** @protected */
   render() {
     return html`
-      <style>
-        vaadin-login-form-wrapper > form > * {
-          width: 100%;
```

Comment:
This is still needed for Lumo, and visual tests passed as we only have them for `vaadin-login-overlay`.

IMO we can also update existing core / Lumo styles to set `display: flex` on the slotted form, as I'd like to remove this `<style>` tag from here. I can create a separate PR for that.